content/governance.md

```diff
@@ -175,9 +175,10 @@ The ex-member is
 
 * removed from the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR. In case of forced removal, no approval is needed.
 * removed from the [GitHub organization][gh] and related organizations and repositories. Optionally, they can retain maintainership of one or more repositories if the [team](#team-members) agrees.
-* removed from the [team mailing list][team].
+* removed from the [team mailing list][team] and demoted to a normal member of the other mailing lists, i.e [developers][devs], [users][], and [announce][]. 
 * announced as removed to CNCF. We will explicitly request reconfirmation of removal from CNCF.
-* removed from shared password storage. All passwords, API tokens, etc are cycled.
+* removed from the shared password storage. All passwords, API tokens, etc. are cycled in due course (i.e. immediately in case of an involuntary departure but can be appropriately batched with other departures or routine cycling otherwise).
```

Comment:
There is still some unclarity how to do the removal concretely, because (apparently) there are two different ways of adding someone to the LastPass Prometheus account:
- The shared password folder can be shared with someone.
- Someone can be added to the Prometheus group (who in turn owns said shared folder).

Personally, I seem to be able to share and unshare the folder with someone, but I don't seem to have access to managing the Prometheus group. We'll clarify with the CNCF how best to do this.

Perhaps @grobie has some context, too, who managed our LastPass account before we got under the umbrella of the CNCF enterprise account.

Reply:
I do actually still have admin access, the only other admin is Chris. I can neither see Max nor Stuart in the group. Shall I add everyone else from the team list?

Reply:
If sharing the shared folder with others works as well (we did that with our most recent member @roidelapluie), perhaps we don't really need to micromanage the group.

I've already put down a note to ask Chris at our next CNCF sync. I guess this is not super urgent to clarify.